cert-manager/overlays/emea/jerry/secrets/clouddns-dns01-emea-svc-acct.enc.yaml

```diff
@@ -0,0 +1,67 @@
+sops:
+    kms: []
+    gcp_kms: []
+    azure_kv: []
+    hc_vault: []
+    age: []
+    lastmodified: "2022-09-06T14:04:21Z"
+    mac: ENC[AES256_GCM,data:nuXmD7KWxut9fr9WdS+Uz7aYOIDNrdQMC4aBKqu2BYSiHvrVXIlEUhyzBreLgT7iJ+hVeR/I0tK8bGV5KmZM026Y6xDArpcwMglZA1gFc+VkUEWTF/dmB1OCb0an2BYTjTTDEAj/PIAES70ISRgGJBC/M7tGqVeO1fjGlISNQ3g=,iv:FZeouY+EQGmQVf3x5PdCjCNYNTVL0563fEzptMQmqdA=,tag:rrZxV8HVGM8fYMu8bSi/AQ==,type:str]
+    pgp:
+        - created_at: "2022-03-08T15:26:08Z"
+          enc: |-
+            -----BEGIN PGP MESSAGE-----
+
+            wcFMA9aKBcudqifiARAAWkW+v07FGVw97cGoyqtRQb2J6werXjLM3eTLhSbGUnVv
+            9R0eRFjFRvcwyaMbCzXE4rkcDQjXtO/j5H/PA7ntSyMO5dU91J6YVIo/e1rS3wAJ
+            mALfynUY3Xzh/Fe2aNToG6Ck3hfLol7MLJrd+FJ7CVLguJaZZ2X1s954YPMsna0a
+            hVhtVKaij/GKv3IoucTTHTQAiqKUMgGfnBjI4sWQJQDPXGbkRVxHEHmrFuz7/VMu
+            T2Z22WXkpy5QOz1WZt03uBjZD9+/9TlVqsvVpamuheupmIyrnOnY1NjqWikEkjLf
+            n5nJcJWsLiAzJ45qk9svR0NZSP/vrF2Tl6LYRGsiyhOfJY8FJV965hE+8w5RKdVq
+            ah4UzqpsNNw4wPL0Ysik15/VRm5koMv1ZSfH70iOLGGyxZk4aBvDyleXr/wlrTzG
+            KRY6V3GAddgLuOwdW32OIX/CZjxaFL49lqhBPhb3oUP8gacm39GBL3CXa9Tik+MA
+            gX+tigru768Si9ydyx/yzjHq6oXiFhP4NsOmRMiRxlGHp50ILnyMtNqyMam0hsfd
+            Pl09yJ9QeSNSxmlBHQZRY17edvMxreeHzjq5b86w1Dg0CtWxcrlbt1rrvKUj80qJ
+            UUVj/SKFSPv8dWVbZ2xhoZsK3syZlO9zFn4TunOr93+O4G1gF7IcQqMCCiimLAjS
+            5gHJzlZlR/eRbz9DKid64gfJUyMnGpZ7FL5u8ipsg0IDKBscdZYmsu1fUini3lkw
+            0qbozmqnEL8GrC5pp8Vmmc3kr6sqR8d0H57/2C0dyROJQuKLCGXAAA==
+            =/osb
+            -----END PGP MESSAGE-----
+          fp: 0508677DD04952D06A943D5B4DC4116D360E3276
+        - created_at: "2022-03-08T15:26:08Z"
+          enc: |-
+            -----BEGIN PGP MESSAGE-----
+
+            wcFMAx7mQRst6P0PARAApe84qIBwx/mg6+c0+uVZZX3rRB8eVwA36XmhTmBuZk0M
+            ooPWzfSZFN+hfYJJg6Z38JeHeJEdBq4lzwBUOSmx8ozOltkQWX1ahzFCP0V9z48B
+            1XVahZgbPF7VvKmoghvrS5WtlIF42urPTQhi8/25xio5cJHglAQys3PaO4jEiSfF
+            ftOvFypwa+nljhG+E3sJCS3xIuKiZtx9RIdSqZmL2B/+i1Uka7J8AHU2POcSUJX2
+            VtWt3SrVR24Cmnih7gYf7CeYG8NaykiIZtNJPnvxS/a8Sb6aGXw5y/a6UBeKjWbu
+            Y07BhWv6GeeY7Y00MGllMZaPUG8nwzKHd5hohR+mz7UMSm/lsINjq8wQOaLcAJv3
+            9wV4ElXd+UHDCypSuc2F5wp5EqNj+NRYVrJ4rgbLYhbyP900lxgk+Y3/Jw6KUmeb
+            ww7GkSDofQdh4JkRUQr8XGWT/wXUEc//T6zb2zN8QfEFNwNvSe2qKv6sFP4TpxU/
+            XOAgtx8fwjwXuBCVjD9GubZ6GMShMKA0LwG5/5s3/+gN7c500LOJLzCnORT/DR9P
+            j1IbGcMeM+2+JDzsHj8G6azleN18f2RcgMpyFsfPHrqO9d6M0j8DbVBzKYBXd8dh
+            /An6/MFVti2sKre1OLy4Eo6emLoPo608xasOhplNHSx8Gw+RQAocJOx4T2qiAM/S
+            5gHrXaGlinJOXJRYJRf3tZUGdgYRdsXFSctLG9+tFuxoLX18cpgyw25z2dDhEK6V
+            qq+HYPrURVTDs5AAu5/gkvXkpDZzim0r5tAGvExFVI6lCuJnO7fzAA==
+            =8/Pb
+            -----END PGP MESSAGE-----
+          fp: 2EB7924D7DA3E2DF4717D3EA9E8653F7D343455A
```

Comment:
which key is this? I recognize the top one as the operate-first key, but not this one...

Reply:
I guess this is the key by @rbo - I'll remove this